### ISSUE TITLE
MacOS support documentation

### ISSUE THREAD

**Issue by 2fd5** [Qarik-Group/disruptor#8]
https://github.com/Qarik-Group/disruptor/blob/c8e18640f3c5e8944e6897cecc40a341d3c4d187/scripts/shell.sh#L5

There is assumption that core-coreutils is installed and in $PATH. Shall we have that documented and if so where can I document it ?

Alternatively we could add a check for `greadlink` in `shell.sh` and display appropriate message...

**Comment by borancar** [comment]
Good point, can easily drop the assumption with a check and helpful message how to install coreutils.
